hera_qm/omnical_metrics.py

```diff
@@ -802,9 +802,10 @@ def plot_chisq_tavg(self, pol_index=0, ants=None, ax=None, save=False, fname=Non
                 fname = os.path.join(self.filedir, fname)
             else:
                 fname = os.path.join(outpath, fname)
-        if custom_ax == False:
+        print('custom ax is : ', custom_ax)
```

Comment:
Is this a debug line? I really don't care if it stays just wonder if it is worth it.